api/envoy/api/v2/auth/cert.proto

```diff
@@ -272,12 +272,17 @@ message UpstreamTlsContext {
   // SNI string to use when creating TLS backend connections.
   string sni = 2 [(validate.rules).string.max_bytes = 255];
 
+  // If true, the SNI from the downstream connection is forwarded to the upstream connection.
+  //
+  // TODO: handle oneof with the sni
+  bool forward_original_sni = 3;
+
   // If true, server-initiated TLS renegotiation will be allowed.
   //
   // .. attention::
   //
   //   TLS renegotiation is considered insecure and shouldn't be used unless absolutely necessary.
-  bool allow_renegotiation = 3;
+  bool allow_renegotiation = 4;
```

Comment:
Is changing the option number for an existing command a breaking change? Can the new option be 4 instead?

Reply:
Right, please don't change field number, use 4 in new field.